Documentation/SiteManagement/Index.rst

```diff
@@ -0,0 +1,62 @@
+..  include:: /Includes.rst.txt
+
+..  _site-management:
+
+===============
+Site Management
+===============
+
+..  _site-configuration:
+
+Create a New Site Configuration
+===============================
+
+Before creating a site configuration, you should have created the root page of
```

Comment:
How do I create the root page of a site? Can we link to somewhere? As this is a "getting started", readers might not know about this.

Reply:
I created an issue and added a todo.